### PR TITLE
v2.1, v3.0, dev: add a warning in batch operations

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### What is changed, added or deleted? 
There is only a warning in transaction for load data. This PR adds a warning in load data page.

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by jackysp** [comment]
PTAL @shenli

**Comment by yikeke** [comment]
**Important Update**: We have added a markdownlint static check in the CI check to improve the quality of our documentation, so your unclosed PR might fail the CI check now.

Click `details` beside the `ci/circleci: lint — Your tests failed on CircleCI` message, and you can check the details for the failure and fix all the issues in your PR. It is recommended to install "markdownlint" extension in your VS Code editor, so it will check all potential issues that fail the CI check. 

@jackysp If you have any problem with troubleshooting, please consult @yikeke for help. Thanks for your cooperation~

Ref: https://github.com/pingcap/docs-cn/pull/1494

**Comment by jackysp** [comment]
Hi @yikeke , it seems you've fixed all the lint issues. PTAL again, thanks!

File: dev/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ set @@global.tidb_distsql_scan_concurrency = 10
 
 这个变量用来设置是否自动切分插入数据。仅在 autocommit 开启时有效。
 当插入大量数据时，可以将其设置为 1，这样插入数据会被自动切分为多个 batch，每个 batch 使用一个单独的事务进行插入。
-该用法破坏了事务的原子性，因此，不建议在生产环境中使用。
+该用法破坏了事务的原子性/隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
该用法破坏了事务的原子性和隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```
这里是“原子性和隔离性”还是“原子性或隔离性”？建议用中文表述，“/”在这意义有点模糊。下面的文件也请做相应修改。

Comment (jackysp) [reply dev/reference/configuration/tidb-server/tidb-specific-variables.md]: "and" is better.

File: dev/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 2]
```diff
@@ -222,7 +222,7 @@ set @@global.tidb_distsql_scan_concurrency = 10
 
 这个变量用来设置是否自动切分待删除的数据。仅在 autocommit 开启，并且是单表删除的 SQL 时有效。关于单表删除的 SQL 的定义，详见[这里](https://dev.mysql.com/doc/refman/8.0/en/delete.html)。
 当删除大量数据时，可以将其设置为 1，这样待删除数据会被自动切分为多个 batch，每个 batch 使用一个单独的事务进行删除。
-该用法破坏了事务的原子性，因此，不建议在生产环境中使用。
+该用法破坏了事务的原子性/隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
该用法破坏了事务的原子性和隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```

File: dev/reference/sql/statements/load-data.md [hunk 3]
```diff
@@ -41,6 +41,10 @@ Records: 815264  Deleted: 0  Skipped: 0  Warnings: 0
 
 * 默认情况下，TiDB 每 20,000 行会进行一次提交。这类似于 MySQL NDB Cluster，但并非 InnoDB 存储引擎的默认配置。
 
+> **注意：**
+>
+> 这种拆分事务提交的方式是以打破事务的原子性/隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```
Comment (yikeke) [inline dev/reference/sql/statements/load-data.md]:
```suggestion
> 这种拆分事务提交的方式是以打破事务的原子性和隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```

File: dev/reference/transactions/transaction-model.md [hunk 4]
```diff
@@ -64,4 +64,4 @@ COMMIT;
 
     > **注意：**
     >
-    > `LOAD DATA` 在 TiDB 中没有事务原子性的保证，可以在数据初次导入时使用，不建议在生产环境中使用。
+    > `LOAD DATA` 在 TiDB 中默认会拆分事务后分批提交，这种拆分事务提交的方式是以打破事务的原子性/隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。不建议在生产环境中使用。
```
Comment (yikeke) [inline dev/reference/transactions/transaction-model.md]:
```suggestion
    > `LOAD DATA` 在 TiDB 中默认会拆分事务后分批提交，这种拆分事务提交的方式是以打破事务的原子性和隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。不建议在生产环境中使用。
```

File: v2.1/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 5]
```diff
@@ -188,7 +188,7 @@ set @@global.tidb_distsql_scan_concurrency = 10
 
 这个变量用来设置是否自动切分插入数据。仅在 autocommit 开启时有效。
 当插入大量数据时，可以将其设置为 1，这样插入数据会被自动切分为多个 batch，每个 batch 使用一个单独的事务进行插入。
-该用法破坏了事务的原子性，因此，不建议在生产环境中使用。
+该用法破坏了事务的原子性/隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```
Comment (yikeke) [inline v2.1/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
该用法破坏了事务的原子性和隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```

File: v2.1/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 6]
```diff
@@ -198,7 +198,7 @@ set @@global.tidb_distsql_scan_concurrency = 10
 
 这个变量用来设置是否自动切分待删除的数据。仅在 autocommit 开启，并且是单表删除的 SQL 时有效。关于单表删除的 SQL 的定义，详见[这里](https://dev.mysql.com/doc/refman/8.0/en/delete.html)。
 当删除大量数据时，可以将其设置为 1，这样待删除数据会被自动切分为多个 batch，每个 batch 使用一个单独的事务进行删除。
-该用法破坏了事务的原子性，因此，不建议在生产环境中使用。
+该用法破坏了事务的原子性/隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```
Comment (yikeke) [inline v2.1/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
该用法破坏了事务的原子性和隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```

File: v2.1/reference/sql/statements/load-data.md [hunk 7]
```diff
@@ -41,6 +41,10 @@ Records: 815264  Deleted: 0  Skipped: 0  Warnings: 0
 
 * 默认情况下，TiDB 每 20,000 行会进行一次提交。这类似于 MySQL NDB Cluster，但并非 InnoDB 存储引擎的默认配置。
 
+> **注意：**
+>
+> 这种拆分事务提交的方式是以打破事务的原子性/隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```
Comment (yikeke) [inline v2.1/reference/sql/statements/load-data.md]:
```suggestion
> 这种拆分事务提交的方式是以打破事务的原子性和隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```

File: v2.1/reference/transactions/transaction-model.md [hunk 8]
```diff
@@ -64,4 +64,4 @@ COMMIT;
 
     > **注意：**
     >
-    > `LOAD DATA` 在 TiDB 中没有事务原子性的保证，可以在数据初次导入时使用，不建议在生产环境中使用。
+    > `LOAD DATA` 在 TiDB 中默认会拆分事务后分批提交，这种拆分事务提交的方式是以打破事务的原子性/隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。不建议在生产环境中使用。
```
Comment (yikeke) [inline v2.1/reference/transactions/transaction-model.md]:
```suggestion
    > `LOAD DATA` 在 TiDB 中默认会拆分事务后分批提交，这种拆分事务提交的方式是以打破事务的原子性和隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。不建议在生产环境中使用。
```

File: v3.0/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 9]
```diff
@@ -214,7 +214,7 @@ set @@global.tidb_distsql_scan_concurrency = 10
 
 这个变量用来设置是否自动切分插入数据。仅在 autocommit 开启时有效。
 当插入大量数据时，可以将其设置为 1，这样插入数据会被自动切分为多个 batch，每个 batch 使用一个单独的事务进行插入。
-该用法破坏了事务的原子性，因此，不建议在生产环境中使用。
+该用法破坏了事务的原子性/隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```
Comment (yikeke) [inline v3.0/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
该用法破坏了事务的原子性和隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```

File: v3.0/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 10]
```diff
@@ -224,7 +224,7 @@ set @@global.tidb_distsql_scan_concurrency = 10
 
 这个变量用来设置是否自动切分待删除的数据。仅在 autocommit 开启，并且是单表删除的 SQL 时有效。关于单表删除的 SQL 的定义，详见[这里](https://dev.mysql.com/doc/refman/8.0/en/delete.html)。
 当删除大量数据时，可以将其设置为 1，这样待删除数据会被自动切分为多个 batch，每个 batch 使用一个单独的事务进行删除。
-该用法破坏了事务的原子性，因此，不建议在生产环境中使用。
+该用法破坏了事务的原子性/隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```
Comment (yikeke) [inline v3.0/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
该用法破坏了事务的原子性和隔离性，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```

File: v3.0/reference/sql/statements/load-data.md [hunk 11]
```diff
@@ -41,8 +41,12 @@ Records: 815264  Deleted: 0  Skipped: 0  Warnings: 0
 
 * 默认情况下，TiDB 每 20,000 行会进行一次提交。这类似于 MySQL NDB Cluster，但并非 InnoDB 存储引擎的默认配置。
 
+> **注意：**
+>
+> 这种拆分事务提交的方式是以打破事务的原子性/隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```
Comment (yikeke) [inline v3.0/reference/sql/statements/load-data.md]:
```suggestion
> 这种拆分事务提交的方式是以打破事务的原子性和隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。因此，不建议在生产环境中使用。
```

File: v3.0/reference/transactions/transaction-model.md [hunk 12]
```diff
@@ -65,4 +65,4 @@ COMMIT;
 
     > **注意：**
     >
-    > `LOAD DATA` 在 TiDB 中没有事务原子性的保证，可以在数据初次导入时使用，不建议在生产环境中使用。
+    > `LOAD DATA` 在 TiDB 中默认会拆分事务后分批提交，这种拆分事务提交的方式是以打破事务的原子性/隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。不建议在生产环境中使用。
```
Comment (yikeke) [inline v3.0/reference/transactions/transaction-model.md]:
```suggestion
    > `LOAD DATA` 在 TiDB 中默认会拆分事务后分批提交，这种拆分事务提交的方式是以打破事务的原子性和隔离性为代价的，使用该特性时，使用者需要保证没有其他对正在处理的表的**任何**操作，并且在出现报错时，需要及时**人工介入，检查数据的一致性和完整性**。不建议在生产环境中使用。
```

**Comment by yikeke** [comment]
There are other changes you need to update. @jackysp And this pull request needs a technical review, thanks~

**Comment by jackysp** [comment]
Addressed, @yikeke !
PTAL @lysu , thanks!

**Pull request merged**